python/src/trezorlib/cli/settings.py

```diff
@@ -210,6 +210,17 @@ def safety_checks(client, always, level):
     return device.apply_settings(client, safety_checks=level)
 
 
+@cli.command()
+@click.option("--enable/--disable", default=False)
```

Comment:
please convert this to an argument with `on/off` possibilities

Reply:
Fixed: 978cc11b07d04eb1a1695bc0c7798eb34eddaff9.